app/assets/stylesheets/spotlight/_header.scss

```diff
@@ -31,112 +28,93 @@ $masthead-image-blur: 1px;
   }
 }
 
-#exhibit-masthead,
-#exhibit-navbar {
-  z-index: 1;
-}
-
-#exhibit-navbar {
-  z-index: 100;
-
-  &.page-masthead {
-    @include transparent-masthead-navigation-menu();
-    background-color: $navbar-transparent-page-bg;
-    border-bottom: 1px solid $navbar-transparent-border;
-    margin-bottom: 0;
-    margin-top: 0;
-
-    .navbar-nav {
-      float: right;
-    }
-  }
-
-  .navbar-form {
-    text-align: right;
-  }
-}
-
-#exhibit-masthead.with-image + #exhibit-navbar.navbar-default {
-  @include transparent-masthead-navigation-menu();
-  background-color: $navbar-transparent-bg;
-  border-top: 1px solid $navbar-transparent-border;
-}
-
-.navbar + .navbar {
-  margin-top: 0;
-}
-
 #header-navbar {
   margin-bottom: 0;
 }
 
-#exhibit-masthead {
-  height: $masthead-height;
-  margin-bottom: -50px;
+.masthead {
+  // allow dropdowns in the top nav to stack above the masthead
+  z-index: $zindex-navbar - 1;
+  margin-bottom: $padding-large-vertical * 2.5;
+  border-bottom: 1px solid $navbar-default-border;
+
+  background-color: $masthead-bg;
   padding: 0;
-  position: static;
+  position: relative;
+  overflow: hidden;
 
-  & > .container {
+  .site-title {
     position: relative;
-    top: 33%;
-    -webkit-transform: translateY(-50%);
-    transform: translateY(-50%);
+    white-space: nowrap;
+    padding-bottom: $padding-base-vertical;
   }
 
-  &.with-page-masthead {
-    margin-bottom: $padding-base-vertical;
-    margin-top: -51px;
+  small {
+    display: block;
+    margin-top: -$padding-base-vertical;
+    padding-top: $padding-base-vertical;
+    padding-bottom: $padding-base-vertical;
+    @extend .hidden-xs;
```

Comment:
Expected item on line 57 to appear before line 53. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets